test/runner/configuration/include_test.dart

```diff
@@ -83,7 +83,30 @@ void main() {
   });
 
   test('should not allow an include field in a test config context', () async {
-    expect(() => new Configuration.parse(['--include=dart_test.yaml']),
-        throwsFormatException);
+    await d.dir('repo', [
+      d.dir('pkg', [
+        d.file('dart_test.yaml', r'''
+          tags:
+            foo:
+              include: ../dart_test.yaml
+        '''),
+      ]),
+    ]).create();
+    var path = p.join(d.sandbox, 'repo', 'pkg', 'dart_test.yaml');
+    expect(() => new Configuration.load(path), throwsFormatException);
+  });
+
+  test('should allow an include field in a runner config context', () async {
+    await d.dir('repo', [
+      d.dir('pkg', [
+        d.file('dart_test.yaml', r'''
+          presets:
+            foo:
+              include: ../dart_test.yaml
+        '''),
+      ]),
+    ]).create();
+    var path = p.join(d.sandbox, 'repo', 'pkg', 'dart_test.yaml');
+    expect(() => new Configuration.load(path), returnsNormally);
```

Comment:
As written, this should fail due to `../dart_test.yaml` not existing.

This test should assert that the preset contains configuration from the corresponding file.

Reply:
Took a bit of refactoring, but done now.